client/src/components/OnboardUser.js

```diff
@@ -0,0 +1,233 @@
+import React, { useState } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import NavBar from './NavBar';
+import { ReactComponent as Logo } from '../img/Logo_CORONATRACKER_Logo.svg';
+import { ReactComponent as TextLogo } from '../img/Logo_CORONATRACKER_Text_Logo.svg';
+import profileImg from '../img/profile.png';
+
+const useStyles = makeStyles(theme => ({
+  //the styles goes here as an object
+  root: {
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'space-evenly',
+    margin: '2em 0 2em 0',
+  },
+  button: {
+    color: 'white',
+    backgroundColor: 'red',
+    fontWeight: 'bold',
+    alignSelf: 'center',
+    height: '4vh',
+    borderRadius: '5px',
+    margin: '0 0 0 0',
+    maxWidth: '300px',
+    minWidth: '42%',
+  },
+  header: {
+    width: 325,
+    display: `inline-block`,
+    marginBottom: '.5em',
+    fontSize: '.5em',
+  },
+  profile: {
+    width: '100px',
+    height: '140px',
+    marginBottom: '.5em',
+    fontSize: '1em',
+  },
+  image: {
+    width: '100%',
+  },
+  inputArea: {
+    display: 'flex',
+    flexDirection: 'row',
+    marginBottom: '.5vh',
+  },
+  inputField: {
+    width: '48%',
+    textAlign: 'center',
+    maxWidth: '180px',
+    color: 'white',
+    backgroundColor: 'red',
+    height: '4vh',
+    justifyContent: 'space-between',
+    border: 'none',
+    borderRadius: '5px',
+    '&::placeholder': {
+      color: 'white',
+    },
+
+    '&:-ms-input-placeholder': {
+      color: 'white',
+    },
+
+    '&::-ms-input-placeholder': {
+      color: 'white',
+    },
+  },
+  inputFieldLabel: {
+    textAlign: 'right',
+    width: '48%',
+  },
+  location: {
+    border: 'none',
+    marginBottom: '10px',
+    textAlign: 'center',
+    color: 'white',
+    backgroundColor: 'red',
+    width: '33%',
+    maxWidth: '60px',
+    fontSize: '1em',
+    borderRadius: '5px',
+    fontFamily:
+      '-apple-system, BlinkMacSystemFont, avenir next, avenir, helvetica neue, helvetica, ubuntu, roboto, noto, segoe ui, arial,',
+    '&::placeholder': {
+      color: 'white',
+    },
+
+    '&:-ms-input-placeholder': {
+      color: 'white',
+    },
+
+    '&::-ms-input-placeholder': {
+      color: 'white',
+    },
+  },
+}));
+
+const blankForm = {
+  firstName: '',
+  age: '',
+  sex: '',
+  city: '',
+  state: '',
+  zip: '',
+};
+
+export default ({ postNewUser }) => {
```

Comment:
Doesn't this component need a name?

Reply:
I can give it a name within the file if you like.  It's a default export so the name in the file doesn't affect the name when you import it.

Reply:
Cool, didn't realize that was a feature. I'll leave it up to you if you want to add one or not, but it may just be useful for consistency's sake then.

Reply:
Fixed.